packages/xo-web/src/xo-app/backup/overview/index.js

```diff
@@ -239,7 +239,12 @@ export default class Overview extends Component {
           <CardBlock>
             <NoObjects
               collection={schedules}
-              emptyMessage={_('noScheduledJobs')}
+              emptyMessage={
+                <span>
+                  {_('noScheduledJobs')}{' '}
+                  <Link to='/backup-ng/health'>{_('legacySnapshotsLink')}</Link>
+                </span>
```

Comment:
Why?

Reply:
So the user knows that he can delete the old snapshot if he'd like to. See [this](https://github.com/vatesfr/xen-orchestra/issues/3082#issuecomment-399012630).